### DISCUSSION TITLE
Unable to rent from tavern

### DISCUSSION THREAD

**Discussion by crabustc** [coccoinomane/crabada.py#83] [Q&A]
Hi guys,

I've setup everything, working fine on windows. 

I got one account running with own reinforcement which is doing great but my other account which require tavern renting has issues. I've tried changing to multiple different reinforcement strategy but the tavern renting doesn't seem to work at all. Am using VM's and windows 10.

Not sure if you guys might have an idea on how to solve this but thank you for your time!

**Comment by coccoinomane** [comment]
Hi @crabustc , and welcome to the community!

Before a wallet can be used on the bot, you have you to reinforce at least once on play.crabada.com; more details here > https://github.com/coccoinomane/crabada.py/discussions/89

Let me know if this fixes your problem :-)

Cheers,
Cocco

**Reply by crabustc** [reply]
hi man,

 i've added metamask to my vm and made a transaction on the tavern manually but still not able to rent from the tavern. 

using python -m bin.looting.reinforceAttack walletaddress

gasfee is on limitless

is there any other modification I need to make before it works? maybe I missed some changes?

![image](https://user-images.githubusercontent.com/104249861/165906830-bd3d9f1d-c043-4fcb-a884-b6e74987e79d.png)
![image](https://user-images.githubusercontent.com/104249861/165906875-c290661d-82f9-4085-a2b0-aac1a0cdecc7.png)

**Comment by coccoinomane** [comment]
Hi @crabustc !

May I ask why is you `REINFORCEMENT_TO_PICK` setting so high? Most times you don't even need the setting.

Also, I don't see the `REINFORCEMENT_MAX_PRICE` setting; I'd recommend setting it to at least 20.

Cheers,
Cocco

**Reply by crabustc** [reply]
thanks for your quick reply.

ive set the reinforcement to pick to 5 now and max price at 25.

![image](https://user-images.githubusercontent.com/104249861/165913367-67a06737-ea81-4075-9b73-80921d52c310.png)


still getting this issue

**Reply by loayei** [reply]
Are you using the latest version of the script?

**Reply by CBSInvest** [reply]
@crabustic
Did u solve the issue cause i get the same error message 
src.common.logger - INFO - Strategy 'HighestBp' could not find a suitable reinforcement for team 13217
src.common.logger - WARNING - NoSuitableReinforcementFound: No suitable reinforcement found [mine=6121675, team=13217]

REINFORCEMENT_MAX_PRICE 23
REINFORCEMENT_TO_PICK  3

Curious which solution u got thx !!!

**Reply by crabustc** [reply]
> @crabustic Did u solve the issue cause i get the same error message src.common.logger - INFO - Strategy 'HighestBp' could not find a suitable reinforcement for team 13217 src.common.logger - WARNING - NoSuitableReinforcementFound: No suitable reinforcement found [mine=6121675, team=13217]
> 
> REINFORCEMENT_MAX_PRICE 23 REINFORCEMENT_TO_PICK 3
> 
> Curious which solution u got thx !!!

Nope didn't find a solution yet

**Comment by CBSInvest** [comment]
@loayei 
Yes downloaded again the latest version and tried to reinforce now for the mining team and not looting team but also here same error message:

src.common.logger - INFO - Strategy 'HighestMp' could not find a suitable reinforcement for team 13217
src.common.logger - WARNING - NoSuitableReinforcementFound: No suitable reinforcement found [mine=6127524, team=13217]
src.common.logger - INFO - Strategy 'HighestMp' could not find a suitable reinforcement for team 15333
src.common.logger - WARNING - NoSuitableReinforcementFound: No suitable reinforcement found [mine=6127516, team=15333]
